### PR TITLE
Feature/update design configs

### PR DESCRIPTION
**Pull request by gwelter**
Refactor colors config to allow usage of tailwind's `opacity` classes. Also start developing the generic button component.

## Changes

- Colors now defined on `index.css` as just rgb number
- Add generic `Button` component that later will be extended but `LoginButton` and `RegisterButton`

## Checklist

- [x] Unit tests
- [x] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "coverage": "jest --collectCoverage --coverageProvider=babel --config ./jest.config.js",
     "test": "jest --no-cache --config ./jest.config.js",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook --no-manager-cache -p 6006",
```
Comment (joshLong145) [inline package.json:20]: is there a reason we removed the caching flag?

File: src/components/Button/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+  return (
+    <div className="inline-flex items-center mx-auto pt-8">
+      <button
+        id={id}
+        type={type}
+        className={styling}
+        onClick={onClick}
+      >
+        {icon && <SonrLogoWrapper />}
```
Comment (joshLong145) [inline src/components/Button/Button.tsx:15]: Does this render the text `false` in cases where the icon is not provided?

**Review by joshLong145 — COMMENTED** [review]
looks good left a few questions

File: src/components/Button/Button.tsx (hunk 2 above)
Comment (gwelter) [reply src/components/Button/Button.tsx:16]: Humm, i'll test that, didn't thougt about this case.

File: package.json (hunk 1 above)
Comment (gwelter) [reply package.json:20]: Sorry, just forgot to remove from the PR.

Comment (gwelter) [reply package.json:20]: Actually no, that's the common way of preventing something from rendering on React. False, null and undefined (not the string) are just not rendered.

File: src/components/Button/Button.tsx (hunk 2 above)
Comment (gwelter) [reply src/components/Button/Button.tsx:16]: Actually no, that's the common way of preventing something from rendering on React. False, null and undefined (not the string) are just not rendered.

Comment (joshLong145) [reply src/components/Button/Button.tsx:16]: yes this is convention, ill resolve.

**Review by joshLong145 — APPROVED** [review]

**Pull request merged**